config/routes.rb

```diff
@@ -6,11 +6,17 @@
       get "/auto_login", to: "users#auto_login"
       get "/users", to: "users#index"
       get "/user/:id", to: "users#show"
+
       resources :motorcycles, only: %i[index]
       get "/motorcycle/:id", to: "motorcycles#show"
       post "/motorcycle", to: "motorcycles#create"
       put "/motorcycle/:id", to: "motorcycles#update"
       delete "/motorcycle/:id => false", to: "motorcycles#destroy"
```

Comment:
- [ ] Please  update this line 
```ruby
delete "/motorcycle/:id => false", to: "motorcycles#destroy"
```
to 
```ruby
delete "/motorcycle/:id", to: "motorcycles#destroy"
```
  - Because it makes the routing impossible to access